cluster/gce/gci/apiserver_manifest_test.go

```diff
@@ -92,7 +92,8 @@ func (c *kubeAPIServerManifestTestCase) invokeTest(e kubeAPIServerEnv, kubeEnv s
 	c.mustLoadPodFromManifest()
 }
 
-func TestEncryptionProviderFlag(t *testing.T) {
+//testcase doesn't work with multiapiserver config, temporally disable
+func testEncryptionProviderFlag(t *testing.T) {
```

Comment:
Please add more details into the comment so that others understand why this needs to be disabled

FYI: you can use t.Skip() to skip a unit test.

Reply:
add comments and updated to use t.Skip()